src/org/intellij/markdown/html/HtmlGenerator.kt

```diff
@@ -48,7 +50,7 @@ class HtmlGenerator(private val markdownText: String,
                                   vararg attributes: CharSequence?,
                                   autoClose: Boolean = false) {
             htmlString.append("<$tagName")
-            for (attribute in attributes) {
+            for (attribute in attributes.asIterable().let { customizer?.invoke(node, tagName, it) ?: it }) {
```

Comment:
Can we reverse this call chain? Like `customizer?.invoke(...) ?: attributes`?

Reply:
Also one might consider making customizer notnull and default it to dummy customizer `(_, _, a) -> a`, that will eliminate elvis here.

Reply:
Ok, will make customizer not-null.

Reply:
> Ok, will make customizer not-null.

I am not 100% sure this is better, leaving that to your judgement :)